src/go/intelligence/models/models.go

```diff
@@ -121,11 +121,17 @@ type silence struct {
 	SilenceStatus []string `json:"silenceStatus"` //Labels for status of the silence
 }
 
+//annotations struct for storing specific set of keywords & set of dashboards to annotate when these keywords are matched in alerts.
+type annotationsData struct {
+	Actions []string `json:"actions"`
+	Systems []string `json:"systems"`
+	Tags    []string `json:"tags"`
+}
+
 type annotationMap struct {
-	Label    string   `json:"label"`
-	Actions  []string `json:"actions"`
-	Systems  []string `json:"systems"`
-	URLLabel string   `json:"urlLabel"`
+	Label           string            `json:"label"`
```

Comment:
add doc strings for each member of the struct

Reply:
added.